index.html

```diff
@@ -511,16 +521,15 @@ <h2>Avant</h2>
             </section>
             <section>
                 <div data-markdown>
-                    ## Ce qu'on a mis en place
+                    ### Image Docker
                     Possibilité de précharger des données à partir d'un dump
-
                 </div>
                 <div data-markdown class="fragment">
 
 ```
 #!/bin/bash
 if [ -f /opt/bonita/dump/bonita.dump ]; then
-psql -h localhost -p 5432 -Ubonita bonita < /opt/bonita/dump/bonita.dump
+psql -h localhost -p 5432 -Uuser bonita < /opt/bonita/dump/bonita.dump
```

Comment:
-Ubonita means user bonita why changing this ?

Reply:
pour que ça tienne sur la largeur de la page

Reply:
on met un `< \`pour satisfaire les puristes ?